### ISSUE TITLE
Compatibility with Panasonic  CZ-CAPWFC1 WiFi interface module ?

### ISSUE THREAD

**Issue by mr-sneezy** [DomiStyle/esphome-panasonic-ac#26]

**Comment by uplore** [comment]
I came here to ask this exact question. I would love to disconnect comfort cloud from our central AC unit. Here is an image of the internals of the CZ-CAPWFC1
![20210829_224812](https://user-images.githubusercontent.com/65812203/132858035-0dc753ac-968c-4d9f-a3df-8368beeb6a4e.jpg)

**Comment by HA-TB303** [comment]
That bottom module looks identical to the DNSK-P11

**Comment by DomiStyle** [comment]
Yep, that's it. You probably can replace the bottom module with an ESP. Make sure to measure voltages on the connector first to be sure. There's a pinout in the DNSK-P11 readme.

**Comment by DomiStyle** [comment]
@vjmaz Take a look at the readme for the DNSK-P11: https://github.com/DomiStyle/esphome-panasonic-ac/blob/master/README.DNSKP11.md

There's a pinout there for the 8 pin connector.

**Comment by rndm2** [comment]
Hi 
Any progress on this?
